host/xtest/Makefile

```diff
@@ -205,6 +204,20 @@ $(foreach v, $(GCM_NIST_VECTORS_ENCRYPT), $(eval $(call \
 	create-nist-gcm-vectors,$v,encrypt)))
 endif
 
+define embed-file
+cleanfiles += $(out-dir)/xtest/$(1).h
+
+$(out-dir)/xtest/$(1).h: $(2)
+	@echo '  GEN     $$@'
+	$(q)../../scripts/file_to_c.py --inf $$< --out $$@ --name $(1)
+
+$(CURDIR)/regression_8100.c: $(out-dir)/xtest/$(1).h
+endef
+
+$(eval $(call embed-file,regression_8100_ca_crt,../../cert/ca.crt))
+$(eval $(call embed-file,regression_8100_mid_crt,../../cert/mid.crt))
+$(eval $(call embed-file,regression_8100_my_crt,../../cert/my.crt))
+$(eval $(call embed-file,regression_8100_my_csr,../../cert/my.csr))
```

Comment:
So the aosp ci detected an actual build regression this time.

```
optee_test/host/xtest/regression_8100.c:23:10: fatal error: 'regression_8100_ca_crt.h' file not found
```

Not sure if we can call a python script in `Android.mk`. @liuyq any ideas?

Or can these header files be pre-generated?